simd/keccakf1600/example_test.go

```diff
@@ -0,0 +1,79 @@
+package keccakf1600_test
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/cloudflare/circl/simd/keccakf1600"
+	"golang.org/x/crypto/sha3"
```

Comment:
Apply this change, so we don't need `x/crypto` in go.mod

```suggestion
	sha3 "github.com/cloudflare/circl/internal/shake"
```